tcn/tcn.py

```diff
@@ -251,7 +238,7 @@ def __init__(self,
         self.output_slice_index = None  # in case return_sequence=False
         self.padding_same_and_time_dim_unknown = False  # edge case if padding='same' and time_dim = None
 
-        if self.use_batch_norm + self.use_layer_norm + self.use_weight_norm > 1:
+        if self.use_batch_norm + self.use_layer_norm > 1:
```

Comment:
I thought to keep my changes minimal, but do we want to turn this into an `or` statement instead? since there's only two variables to check now